### DISCUSSION TITLE
Assignments Submission to be eligible for receiving ESIIL badge

### DISCUSSION THREAD

**Discussion by DharaniSureshBabu** [cu-esiil-edu/2024-data-short-course#7] [Q&A]
@eculler @nquarder I have a quick question regarding the submission of assignments. Could you please guide me on where to submit them? I've tried creating a new GitHub markdown webpage, https://dharanisureshbabu.github.io/  but I'm unsure where to submit this to become eligible for receiving the reputed ESIIL badge.

**Comment by nquarder** [comment]
Hi @DharaniSureshBabu and great job on your page! We did go through this in office hours today and got a recording of it. I am sharing a link [here](https://cuboulder.zoom.us/rec/share/--A9UC5zZSOi3peoqVzEEETNoj_lE48Q9hAZVUZN9UYGA-SY6lV1lf9TWHgeoz7J.IpJTyozDG3-B97ud) to the recording for you and others to see. If you skip to the last 30 min or so you will see us demonstrating it with @gauravchakrabarty , @Garcia-INPE , @jarrodredbird and @chyasirabbasjutt !

I will also do my best to explain this process below for anyone who can't watch the recording. 

1. From this `2024-data-short-course` repository (that's the one we're working in now!) you want to select the `.index.md` file.
2. Go to the little pencil icon in the upper right corner to edit the file.
3. You will be prompted to `Fork this repository and edit the file`.
4. Go ahead and fork the repository (green `Fork this repository` button). This will make a copy that you will own and will be able to make changes to.
5. You'll see a message that says `You’re making changes in a project you don’t have write access to. Submitting a change will write it to a new branch in your fork <username>/2024-data-short-course, so you can send a pull request.` Below that you'll see the text editor where you'll now be able to make edits to the `index.md` file.
6. Scroll down towards the bottom to where you see the heading that says `Participants`. Here you will see a table written using Markdown syntax where we are asking people to add their _name_, _organization_, _GitHub username_, and _**link** to your username.github.io personal portfolio page_. These will be separated by pipes (vertical bars) as shown in the example below
> **EXAMPLE:**   | Name      | Organization | GitHub Username |  GitHub Portfolio Link    |      
7. We'd love to hear how folks would like their names pronounced. To do this you can create an account with a tool called namedrop (link - [https://namedrop.io/](https://namedrop.io/)). Once you have an account you can record yourself saying your name and it will create a link that you can share with others. Try linking to your namedrop pronunciation when you add your name to the table. 
8. Once you're done adding your information to the `index.md` you'll want to go back up to the top and hit the green `Commit Changes` button. You'll be prompted to leave yourself a message and then hit the green `Propose changes` button.
9. You'll then be prompted to submit what is called a `Pull Request`. Select the green `Create pull request` button, leave a message and tag @eculler or @nquarder so that we get notified of your request and that we can merge your changes. 

I hope this helps @DharaniSureshBabu and let us know if you run into any issues with this last step. Again, great job on your portfolio page! -NQ
